web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -135,7 +119,7 @@ const ReviewPeriods = ({ onPeriodSelected, mode }) => {
     closeDate: null
   });
   const [reviewStatus, setReviewStatus] = useState(ReviewStatus.CLOSED);
-  const [selfReviews, setSelfReviews] = useState(null);
+  const [selfReviews, setSelfReviews] = useState({});
```

Comment:
This holds a map of review data. Initializing it to an empty object simplifies the code a bit.